.github/workflows/scripts/slack_gem_notifications.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require 'httparty'
+
+# TODO: Add support for comparing latest and previous ruby versions
+# TODO: If possible, specify how many gem objects we recieve from Ruby Gems API. We don't
+# need a gem's entire history.
+
+WATCHED_GEMS = ARGV
+
+if WATCHED_GEMS.empty?
+  warn 'WATCHED_GEMS cannot be empty'
+  exit
+end
+
+def watch_gem(gem_name)
+  gem_info = HTTParty.get("https://rubygems.org/api/v1/versions/#{gem_name}.json")
+  latest_versions = []
+  gem_info.each do |gem|
+    latest_versions << gem if gem['platform'] == 'ruby'
+    break if latest_versions.size == 2
+  end
+  return unless Time.now.utc - Time.parse(latest_versions[0]['created_at']) < 24 * 60 * 60
+
+  compare_gems(gem_name, latest_versions)
+end
+
+def compare_gems(gem_name, latest_versions)
+  newest, previous = latest_versions[0]['number'], latest_versions[1]['number']
+  send_bot(gem_name, previous, newest)
+end
+
+def send_bot(gem, current_version, latest_version)
+  `curl -X POST -H 'Content-type: application/json' --data '{"text":"Hey Ruby team! There is a new gem version release.
```

Comment:
After the work done to bring in HTTParty, perhaps it could be used instead of backticking `curl` here to do this POST.

As-is, the `curl` command is problematic, because on a multi-user unix system, the publicly available process list will display the sensitive webhook environment variable. This is possibly not an issue with GHA, but would definitely be addressed with a switch to HTTParty.